testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oid4vc/issuance/signing/JWTVCIssuerWellKnownProviderTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2024 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.keycloak.testsuite.oid4vc.issuance.signing;
+
+import org.junit.Test;
+import org.keycloak.common.util.MultivaluedHashMap;
+import org.keycloak.jose.jwk.JSONWebKeySet;
+import org.keycloak.protocol.oid4vc.issuance.JWTVCIssuerWellKnownProvider;
+import org.keycloak.protocol.oid4vc.model.JWTVCIssuerMetadata;
+import org.keycloak.representations.idm.RealmRepresentation;
+
+import java.util.List;
+import java.util.Map;
+
+import static junit.framework.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+
+/**
+ *
+ * @author <a href="mailto:francis.pouatcha@adorsys.com">Francis Pouatcha</a>
+ */public class JWTVCIssuerWellKnownProviderTest extends OID4VCTest {
+
+     @Test
+    public void getConfig() {
+        String expectedIssuer = suiteContext.getAuthServerInfo().getContextRoot().toString() + "/auth/realms/" + TEST_REALM_NAME;
+        testingClient
```

Comment:
Instead of using `testingClient` here (which is probably the reason of failing test btw), can this be tested by actually sending the request to the endpoint and check that data are really available under specified URL?

For the inspiration, you can take a look at how this is tested for OIDC endpoint. There are methods `AbstractWellKnownProviderTest.getOIDCDiscoveryRepresentation` and `AbstractWellKnownProviderTest.getOIDCDiscoveryConfiguration`, which are called for instance from `OIDCWellKnownProviderTest.testDefaultProviderCustomizations()`

Reply:
Thanks for the hints. Fixed with [4e6c7ec](https://github.com/keycloak/keycloak/pull/29635/commits/4e6c7ec61f01d5ec18180a7fde302d27ca554c30).